General_gNMI/SubscribeLibrary.py

```diff
@@ -200,3 +203,22 @@ def check_sample_updates(self, period: int, count: int, timeout: int) -> None:
             while not initial_tree.covered_by(sample_tree):
                 response = next(next_responses)
                 apply_response(sample_tree, response, UpdateType.STRUCTURE)
+
+    def check_updates_no_aggr(self, timeout: int, encoding: str) -> None:
+        try:
+            for response in self.requester.raw_responses(timeout):
+                if not response.sync_response:
+                    if encoding == gnmi_pb2.Encoding.JSON_IETF:
```

Comment:
join nested `if`s into single `if A and B`...

Reply:
More encodings  may be supported in the future ....